data-prepper-plugins/geoip-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/geoip/GeoIpPrepperConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.amazon.dataprepper.plugins.prepper.geoip;
+
+import com.amazon.dataprepper.plugins.prepper.geoip.provider.DataSource;
+
+public final class GeoIpPrepperConfig {
+    //TODO add config variables for cache size, location fields, possibly more
+    public static final String TARGET_FIELD = "target_field";
+    public static final String DATA_SOURCE = "data_source";
+    public static final String DEFAULT_DATA_SOURCE = DataSource.MaxMindGeolite2CityDatabase.toString();
```

Comment:
Unused? Probably doesn't make sense to have a default source anyway, as they all will require some kind of setup.